### DISCUSSION TITLE
Cannot load environment on React Native

### DISCUSSION THREAD

**Discussion by ninjz** [pmndrs/drei#1728] [Q&A]
I want to add an environment to my 3d scene in React Native as it makes the model looks much better however whenever I try to load it I get an error saying that the `.hdr` file is not found when I use a preset like this:

```jsx         
 <Environment preset="sunset" background={false} />
```

I wonder if it's possible to load my own HDR file as well, but I'm unsure to do that either. I'd much prefer if I can just use one of the presets already defined.

**Comment by alex-marinov** [comment]
This worked for me ok;
```javascript
  <Environment files="/3d/warehouse.hdr" ground={{ height: 20, radius: 200, scale: 5 }}  />
```
